src/chapters/workshop2/aws-cli-ecr/push-to-ecr.js

```diff
@@ -0,0 +1,105 @@
+export const ecrSetupChapter = {
+  id: "cli-ecr",
+  title: "Pushing Docker Image to AWS ECR",
+  sectionId: "aws-cli-ecr",
+  previousChapterId: null,
+  content: `## Configuring AWS CLI with Session Token and Pushing Docker Image to ECR
+
+In this guide, you'll learn how to configure your AWS CLI using temporary credentials (session token), create an Amazon ECR (Elastic Container Registry) repository, and push your Docker image to it.
+
+### 1. Configure AWS CLI Using Session Token
+
+1. Configure your AWS CLI profile
+
+    \`\`\`bash
+    aws configure sso
+    \`\`\`
+
+Follow the prompts to set up and name your profile (e.g., \`nss-sso\`). The region will be \`us-east-2\` and the output format will be \`json\`.
+
+**What's happening here?** This is creating a local file \`~/.aws/config\` with configurations that the cli will use when accessing AWS resources.
+
+
+2. Use the AWS CLI to assume the intro_to_cloud role
+
+   \`\`\`bash
+   aws sts assume-role \
+     --role-arn arn:aws:iam::[your-account-id]:role/intro_to_cloud \
+     --role-session-name tempSession \
+     --profile [your-sso-profile-name]
+   \`\`\`
+
+Get your account id by clicking your username in the top right hand corner in the AWS console to replace \`[your-account-id]\`. For \`[your-sso-profile-name]\` use the same profile name you used during the sso configuration.
+
+3. Copy the \`AccessKeyId\`, \`SecretAccessKey\`, and \`SessionToken\` from the output.
+
+4. Set them as environment variables:
+   \`\`\`bash
+   export AWS_ACCESS_KEY_ID="[AccessKeyId]"
+   export AWS_SECRET_ACCESS_KEY="[SecretAccessKey]"
+   export AWS_SESSION_TOKEN="[SessionToken]"
+   export AWS_DEFAULT_REGION="us-east-2"
+   \`\`\`
+
```

Comment:
You may want to mention here that if they don't want to set these variables every time they start a new terminal session, they can store them long-term in their `~/.zshrc` file on a Mac, or on Windows, via the **Environment Variables** pane in the system settings.

Reply:
With the session token the credentials are temporary anyway. so i think its not worth the extra step. To store long term credentials, aws configure command automatically creates a .aws/credentials file for storing long term locally.